### PR TITLE
Reduces cost of Shield Generator circuits

### PR DESCRIPTION
**Pull request by SonixApache**
From 
MAT_GLASS = 2000, SACID = 20, MAT_PLASMA = 10000, MAT_DIAMOND = 5000, MAT_GOLD = 10000
to 
MAT_GLASS = 2000, SACID = 20, MAT_DIAMOND = 2000, MAT_GOLD = 2000

Price of capacitors also reduced.
Diamond ensures they won't be TOO common (since you need T3 parts to use them anyway)

Tested.

:cl:
 - tweak: Lowered costs of shield generator circuitry

### CODE REVIEW

**Review by PJB3005 — APPROVED** [review]

**Comment by SonixApache** [comment]
resolved conflicts

File: code/modules/research/designs/boards/machine_engie.dm [hunk 1]
```diff
@@ -252,14 +252,15 @@
 	id = "shield_gen"
 	req_tech = list(Tc_BLUESPACE = 4, Tc_PLASMATECH = 3)
 	build_type = IMPRINTER
-	materials = list(MAT_GLASS = 2000, SACID = 20, MAT_PLASMA = 10000, MAT_DIAMOND = 5000, MAT_GOLD = 10000)
+	materials = list(MAT_GLASS = 2000, SACID = 20, MAT_DIAMOND = 2000, MAT_GOLD = 2000)
 	build_path = "/obj/machinery/shield_gen"
+	materials = list(MAT_GLASS = 2000, SACID = 20, MAT_PLASMA = 10000, MAT_DIAMOND = 5000, MAT_GOLD = 10000)
```
Comment (Intigracy) [inline code/modules/research/designs/boards/machine_engie.dm]: wow bill, your mom lets you have TWO materials vars?

Comment (Intigracy) [inline code/modules/research/designs/boards/machine_engie.dm]: Gee Bill, your mom lets you have TWO materials vars?

**Review by Intigracy — CHANGES_REQUESTED** [review]

File: code/modules/research/designs/boards/machine_engie.dm (hunk 1 above)
Comment (SonixApache) [reply code/modules/research/designs/boards/machine_engie.dm]: how does this shit slip under the radar

**Pull request merged**